apps/gpc-verifier/.env.example

```diff
@@ -0,0 +1,5 @@
+NEXT_PUBLIC_GPC_PROVER_CLIENT_URL=
```

Comment:
These need explanatory comments, as well as ideally working default values to minimize setup steps.